---
layout: archive
title: "Talks and presentations"
permalink: /talks/
author_profile: true
---

<!-- {% if site.talkmap_link == true %}

<p style="text-decoration:underline;"><a href="/talkmap.html">See a map of all the places I've given a talk!</a></p>

{% endif %}

{% for post in site.talks reversed %}
  {% include archive-single-talk.html %}
{% endfor %} -->


<!-- ## [EMBO | EMBL Symposium](https://www.embl.org/about/info/course-and-conference-office/events/ees23-10/?utm_source=annualposter&utm_medium=poster&utm_id=EES23-10), EMBO | EMBL Symposium | The non-coding genome, 2023/10/11-2023/10/14
### Title
Transcriptome-wide identification of substrates of the Drosophila RNA export factor Exportin5
### Abstract
Compartmentalization of eukaryotic cells by various membranes necessitates mechanisms that transport biological materials from the sites of production to the sites of usage. Transcripts encoded by the nuclear genome need to be exported from the nucleus following the completion of nuclear processing steps. To export various classes of RNAs, eukaryotes have multiple transport machineries, each of which is specialized in transporting specific RNA cargoes. Exportin5 (Exp-5) is the major microRNA (miRNA) nuclear export factor and recognizes structural features of pre-miRNA hairpins generated through RNA cleavage by Drosha, while it also exports other minihelix-containing RNAs. Surprisingly, clustered pri-miRNAs are also recognized and bound by Exp-5 in human. In Drosophila, Exp5 is suggested to play a major role in tRNA export, where the gene encoding a homolog of the canonical tRNA export factor Exportin-t is missing in its genome. We used the CLIP (crosslinking and immunoprecipitation) technology to understand Exp-5/RNA interactome in the Drosophila cell line S2R+. The CLIP experiment captured known substrates such as tRNAs and miRNAs, and detected candidates of novel Exp-5 substrates including various mRNAs and long non-coding RNAs. This result suggested that Drosophila Exp-5 may have a wider variety of substrates than initially thought. Moreover, Exp5-CLIP reads often contained sequences corresponding to the flanking 5’- and 3’- regions known as leaders and trailers of tRNAs. This observation was unexpected because tRNAs were thought to be exported after the removal of the leader and trailer sequences. In summary, our results provide a genome-wide list of Exp5 substrate candidates, and may suggest a molecular basis to investigate whether Drosophila can selectively export pre-tRNAs after the removal of the flanking sequences. 
### <img src='/files/talk_pre/oraltalk.jpg'> -->


<h2><a href='https://www.linkedin.com/feed/update/urn:li:activity:7125390487204581376/'>EMBO | EMBL Symposium | The non-coding genome</a>,   
  
  Heidelberg, Germany. 2023/10/11-2023/10/14</h2>
<h3>Title</h3>
<p>Transcriptome-wide identification of substrates of the Drosophila RNA export factor Exportin5</p>
<h3>Abstract</h3>
<p>Compartmentalization of eukaryotic cells by various membranes necessitates mechanisms that transport biological materials from the sites of production to the sites of usage. Transcripts encoded by the nuclear genome need to be exported from the nucleus following the completion of nuclear processing steps. To export various classes of RNAs, eukaryotes have multiple transport machineries, each of which is specialized in transporting specific RNA cargoes.   
Exportin5 (Exp-5) is the major microRNA (miRNA) nuclear export factor and recognizes structural features of pre-miRNA hairpins generated through RNA cleavage by Drosha, while it also exports other minihelix-containing RNAs. Surprisingly, clustered pri-miRNAs are also recognized and bound by Exp-5 in human. In Drosophila, Exp5 is suggested to play a major role in tRNA export, where the gene encoding a homolog of the canonical tRNA export factor Exportin-t is missing in its genome. We used the CLIP (crosslinking and immunoprecipitation) technology to understand Exp-5/RNA interactome in the Drosophila cell line S2R+. The CLIP experiment captured known substrates such as tRNAs and miRNAs, and detected candidates of novel Exp-5 substrates including various mRNAs and long non-coding RNAs. This result suggested that Drosophila Exp-5 may have a wider variety of substrates than initially thought. Moreover, Exp5-CLIP reads often contained sequences corresponding to the flanking 5’- and 3’- regions known as leaders and trailers of tRNAs. This observation was unexpected because tRNAs were thought to be exported after the removal of the leader and trailer sequences.  
In summary, our results provide a genome-wide list of Exp5 substrate candidates, and may suggest a molecular basis to investigate whether Drosophila can selectively export pre-tRNAs after the removal of the flanking sequences. 
<img src='/files/talk_pre/oraltalk.jpg'>

