---
layout: archive
title: "Talks and presentations"
permalink: /talks/
author_profile: true
---

<!-- {% if site.talkmap_link == true %}

<p style="text-decoration:underline;"><a href="/talkmap.html">See a map of all the places I've given a talk!</a></p>

{% endif %}

{% for post in site.talks reversed %}
  {% include archive-single-talk.html %}
{% endfor %} -->


<!-- ## [Seminar on Frontiers of Mathematics and Interdisciplinarity](https://xiaoyazhai.github.io/seminar/SFMI1.html), University of Science and Technology of China, 2023/07/08
### Title
Unraveling the Mathematical Conundrum in 4D Printing: A Gateway to the Future of Soft Robotics
### Abstract
This seminar aims to provide a comprehensive overview of the mathematical challenges in 4D printing, their implications, and the potential they hold in advancing soft robotics. The presentation delves into the underpinnings of 4D printing and the intricate mathematical quandaries it introduces, which lie at the heart of the novel functionalities this technology offers. Concurrently, the burgeoning field of soft robotics will be drawn parallelly, highlighting the transformative role 4D printing could play in this domain. 4D printing, an advanced variation of 3D printing, incorporates time as the fourth dimension. It enables the creation of materials that can change shape or properties over time in response to specific stimuli. The mathematical complexities involved in predicting and controlling these transformative behaviors will be dissected. Utilizing a range of mathematical tools such as differential geometry, topology, and non-linear dynamics, I will provide insights into the mechanisms that facilitate the seamless transition between multiple states in 4D printed materials. Simultaneously, the tremendous potential of 4D printing will be illustrated in the realm of soft robotics. As we stand on the brink of a new era where soft robots, with their inherent flexibility and adaptability, are poised to revolutionize industries from healthcare to space exploration, understanding the potential and challenges of 4D printing becomes imperative. Various applications of soft robots will be demonstrated, focusing on how the use of 4D-printed components can enhance their performance, adaptability, and autonomy.
<img src='/files/Essay/tahouni2023codesign.jpg'>
### [Presentation Materials](http://Liuchao-JIN.github.io/files/Essay/tahouni2023codesign.pdf) -->

<h2><a href='https://xiaoyazhai.github.io/seminar/2023/SFMI1.html'>Seminar on Frontiers of Mathematics and Interdisciplinarity</a>, University of Science and Technology of China, 2023/07/08</h2>
<h3>Title</h3>
<p>Transcriptome-wide identification of substrates of the Drosophila RNA export factor Exportin5</p>
<h3>Abstract</h3>
<p>Compartmentalization of eukaryotic cells by various membranes necessitates mechanisms that transport biological materials from the sites of production to the sites of usage. Transcripts encoded by the nuclear genome need to be exported from the nucleus following the completion of nuclear processing steps. To export various classes of RNAs, eukaryotes have multiple transport machineries, each of which is specialized in transporting specific RNA cargoes. 
Exportin5 (Exp-5) is the major microRNA (miRNA) nuclear export factor and recognizes structural features of pre-miRNA hairpins generated through RNA cleavage by Drosha, while it also exports other minihelix-containing RNAs. Surprisingly, clustered pri-miRNAs are also recognized and bound by Exp-5 in human. In Drosophila, Exp5 is suggested to play a major role in tRNA export, where the gene encoding a homolog of the canonical tRNA export factor Exportin-t is missing in its genome. We used the CLIP (crosslinking and immunoprecipitation) technology to understand Exp-5/RNA interactome in the Drosophila cell line S2R+. The CLIP experiment captured known substrates such as tRNAs and miRNAs, and detected candidates of novel Exp-5 substrates including various mRNAs and long non-coding RNAs. This result suggested that Drosophila Exp-5 may have a wider variety of substrates than initially thought. Moreover, Exp5-CLIP reads often contained sequences corresponding to the flanking 5’- and 3’- regions known as leaders and trailers of tRNAs. This observation was unexpected because tRNAs were thought to be exported after the removal of the leader and trailer sequences.
In summary, our results provide a genome-wide list of Exp5 substrate candidates, and may suggest a molecular basis to investigate whether Drosophila can selectively export pre-tRNAs after the removal of the flanking sequences. 
<img src='/files/talk_pre/oraltalk.jpg'>
